### ISSUE TITLE
[Suggestion] Bulk Import (not ini, but class names)

### ISSUE THREAD

**Issue by VasVadum** [thommcgrath/Beacon#44]
Instead of copying and pasting the 63 lines I need to import, I'll just give you the first three;

PrimalItemStructure_Foundation_Wood_C
PrimalItemStructure_TriFoundation_Wood_C
PrimalItemStructure_Wall_Wood_C

It would be nice if instead of inputting one line at a time, I could paste a list of class names in and it auto check the whole list for me of anything with that exact class name.  It is the user's responsibility to ensure that all of the class names he put in, are correct, because any typos are on him to fix.

Basically, when you hit + to add a new item to a set, have a button you can click that shows a multi-line input box, and then you can paste one entry per line so you can copy lines from something such as a spreadsheet right in and hit "check all" or whatever on it to select all of those items and move to the next page as if you had checked a few items and hit next normally.
